fixtures/features/api-elements/extensions.json

```diff
@@ -14,6 +14,26 @@
       "attributes": {},
       "content": [
         {
+          "element": "extension",
+          "meta": {
+            "links": [
+              {
+                "element": "link",
+                "meta": {},
+                "attributes": {
+                  "relation": "profile",
+                  "href": "http://swagger.io/specification/#vendorExtensions"
```

Comment:
Ultimately, I think both solutions would work fine IMO.

However, I think the benefit you would get from using an Apiary-controlled domain as the profile is that you could add attributes in the future for whatever reason you saw fit and it would point consumer to the profile for understanding the entire element rather than just the content. It would also not break previous clients relying on that profile.

Another precedent is that we have always used Apiary domains as far as I know. Unless there would be a compelling reason to not do that, you may want to stick with it.

Lastly—I'm thinking out loud here—but an origin link might be helpful to tell the consumer from what part of a Swagger document the extensions were derived from. In that situation, it would make sense to me for that link to be the Swagger document. Please take this paragraph as thinking out loud :)

Hope that helps some.